mvc/views/view-components.rst

```diff
@@ -21,54 +30,98 @@ New to ASP.NET MVC 6, view components are similar to partial views, but they are
  
 One use of a view component could be to create a login panel that would be displayed on every page with the following functionality:
 
-- If the user is not logged in, a login panel is rendered.
-- If the user is logged in, links to log out and manage account are rendered.
-- If the user is in the admin role, an admin panel is rendered.
+- If the user is not logged in, a login panel is rendered
+- If the user is logged in, links to log out and manage account are rendered
+- If the user is in the admin role, an admin panel is rendered
+
+The view component ``User`` property gives you access to a user's claims, so you can render data depending on the user's claims. You can add this view component view to the layout page and have it render user-specific data throughout the whole application. View components don’t use model binding, and only depend on the data you provide when calling into it. 
+
```

Comment:
This all seems like a lot of explanation for a bullet point in a list of examples. Can we just list "login panel" and move on?